docs/reference/commandline/network_ls.md

```diff
@@ -12,7 +12,7 @@ parent = "smn_cli"
 
     Usage:  docker network ls [OPTIONS]
 
-    Lists all the networks created by the user
+    List networks
```

Comment:
I can change for consistency. My correction was that this lists all networks, not just user-created networks.